src/main/java/frc/robot/RobotContainer.java

```diff
@@ -89,4 +75,26 @@ private void configureButtonBindings() {
     public Command getAutonomousCommand() {
         return autonChooser.getSelected();
     }
+
+    void periodic(){
+        tank.forwardComponent = -1 * controller.getLeftY(); // 1 is forward (adjusted from -1 forward)
+        tank.sideComponent = controller.getRightX(); // 1 is right
+
+        if(controller.getBButton()){ //engage the intake arm by pressing B on the controller
+            intake.intake_down = true;
+
+        }
+        if(controller.getXButton()){
```

Comment:
be wary that this method will return true every loop that the button is held... so if the button is held for more than a split second (whether intentionally or not), this might call more times than you'd like